File: hookInit/oc/oc/ViewController.m
//
//  ViewController.m
//  oc
//
//  Created by Jz D on 2022/5/29.
//

#import "ViewController.h"



@interface ViewController ()

@end

@implementation ViewController


- (void)viewDidLoad{
    [super viewDidLoad];
    
    UICollectionView * upList = [[UICollectionView alloc] initWithFrame: CGRectMake(50, 50, 150, 150) collectionViewLayout: [[UICollectionViewFlowLayout alloc] init]];
    [self.view addSubview: upList];
    
    
}



@end

File: hookInit/swift/swift/ViewController.swift
//
//  ViewController.swift
//  swift
//
//  Created by Jz D on 2022/5/29.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        let v_v = UICollectionView(frame: CGRect(x: 50, y: 50, width: 120, height: 120), collectionViewLayout: UICollectionViewFlowLayout())
        view.addSubview(v_v)
        
    }


}

